crates/polars-core/src/series/implementations/dates_time.rs

```diff
@@ -172,7 +172,8 @@ macro_rules! impl_dyn_series {
             }
 
             fn mean(&self) -> Option<f64> {
-                self.0.mean()
+                let s = self.0.mean();
+                s
```

Comment:
This syntax is less readable than the original, I'd recommend reverting this.

Reply:
Ahh yes, I did this during debugging and did not intend to keep this.